backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -5718,4 +5713,94 @@ databaseChangeLog:
                    (gen_random_uuid(), 'Chlamydia', 'LP14298-1')
       rollback:
         - sql:
-            sql: DELETE FROM ${database.defaultSchemaName}.supported_disease WHERE name = 'Chlamydia';
\ No newline at end of file
+            sql: DELETE FROM ${database.defaultSchemaName}.supported_disease WHERE name = 'Chlamydia';
+  - changeSet:
```

Comment:
do we want a rollback on this to drop this table?

Reply:
createTable (also the addColumn changeset) can be [auto-rolled back](https://docs.liquibase.com/workflows/liquibase-community/automatic-custom-rollbacks.html) so we don't have to write custom rollback statements for them

Reply:
ooo right 🤦🏻 thank you for the reminder